packages/xo-web/src/xo-app/xoa/support/index.js

```diff
@@ -46,6 +47,17 @@ const Support = decorate([
           </Col>
         </Row>
       )}
+      <Row className='mb-1'>
+        <Col>
+          <ActionButton
+            btnStyle='primary'
+            handler={reportOnSupportPanel}
```

Comment:
This button can also be used by the community?

Reply:
No, only for XOA.
See #4234

Reply:
I don't see a restriction for community users. And `reportBugButton` do the same think. You can improve it and let it takes a title as props

Reply:
If I use `reportBugButton`, I should update` tooltip`, `icon`.
The name of the button says "bug", so the tooltip and the icon should only be for the bug.
I also think it's not necessary to use it as it's only intended for repoting bug .

So, the necessary thing is to use `reportOnSupportPanel` to attach files.
What do you think?